endpointmanager/README.md

```diff
@@ -192,4 +184,23 @@ To run, perform the following commands:
 ```bash
 cd endpointmanager/cmd/nppesorgpopulator
 go run main.go <path to nppes csv file>
-```
\ No newline at end of file
+```
+
+### Expected Endpoint Source Formatting
+
+The Network Statistics Querier expects the format of an endpoint source list to be in the below format, unless one of the exceptions noted below.
```

Comment:
Not sure if this should say Network Statistics Querier here since the Network Stats Querier was removed